File: Java/Java/src/PayPal.java
public class PayPal extends Payment{
    String email;

    public PayPal(Integer id, String email){
        super(id);
        this.email = email;
    }

    public void printPayment(){
        System.out.println("Id payment: " + id + " Email: " + email);
    }
}

File: JS/PayPal.js
class PayPal extends Payment {
    constructor(id, email, amount) {
        super(id, amount)
        this.email = email;
    }

    printPayment() {
        console.log("Id payment: " + this.id + " Amount: " + this.amount + " Who deposit: " + this.email);
    }
}